core/templates/dev/head/pages/learner_dashboard/LearnerDashboard.js

```diff
@@ -16,84 +16,91 @@
-});
-
-oppia.constant('FEEDBACK_THREADS_SORT_BY_KEYS', {
-  LAST_UPDATED: 'last_updated',
-  EXPLORATION: 'exploration'
-});
-
-oppia.constant('HUMAN_READABLE_EXPLORATIONS_SORT_BY_KEYS', {
-  LAST_PLAYED: 'I18N_LEARNER_DASHBOARD_EXPLORATIONS_SORT_BY_LAST_PLAYED',
-  TITLE: 'I18N_DASHBOARD_EXPLORATIONS_SORT_BY_TITLE ',
-  CATEGORY: 'I18N_DASHBOARD_EXPLORATIONS_SORT_BY_CATEGORY'
+oppia.constant('EXPLORATIONS_SORT_BY_KEYS_AND_I18N_IDS', {
+  LAST_PLAYED: {
+    key: 'last_played',
+    i18nId: 'I18N_LEARNER_DASHBOARD_EXPLORATIONS_SORT_BY_LAST_PLAYED'
+  },
+  TITLE: {
+    key: 'title',
+    i18nId: 'I18N_DASHBOARD_EXPLORATIONS_SORT_BY_TITLE'
+  },
+  CATEGORY: {
+    key: 'category',
+    i18nId: 'I18N_DASHBOARD_EXPLORATIONS_SORT_BY_CATEGORY'
+  }
 });
 
-oppia.constant('HUMAN_READABLE_SUBSCRIPTION_SORT_BY_KEYS', {
-  USERNAME: 'Username',
-  IMPACT: 'Impact'
+oppia.constant('SUBSCRIPTION_SORT_BY_KEYS_AND_I18N_IDS', {
+  USERNAME: {
+    key: 'subscriber_username',
+    i18nId: 'I18N_PREFERENCES_USERNAME'
+  },
+  IMPACT: {
+    key: 'subscriber_impact',
+    i18nId: 'I18N_CREATOR_IMPACT'
+  }
 });
 
-oppia.constant('HUMAN_READABLE_FEEDBACK_THREADS_SORT_BY_KEYS', {
-  LAST_UPDATED: 'I18N_DASHBOARD_EXPLORATIONS_SORT_BY_LAST_UPDATED',
-  EXPLORATION: 'I18N_DASHBOARD_TABLE_HEADING_EXPLORATION'
+oppia.constant('FEEDBACK_THREADS_SORT_BY_KEYS_AND_I18N_IDS', {
+  LAST_UPDATED: {
+    key: 'last_updated',
+    i18nId: 'I18N_DASHBOARD_EXPLORATIONS_SORT_BY_LAST_UPDATED'
+  },
+  EXPLORATION: {
+    key: 'exploration',
+    i18nId: 'I18N_DASHBOARD_TABLE_HEADING_EXPLORATION'
+  }
 });
 
 oppia.controller('LearnerDashboard', [
   '$scope', '$rootScope', '$window', '$http', '$modal',
-  'EXPLORATIONS_SORT_BY_KEYS', 'SUBSCRIPTION_SORT_BY_KEYS',
-  'HUMAN_READABLE_EXPLORATIONS_SORT_BY_KEYS', 'FATAL_ERROR_CODES',
-  'HUMAN_READABLE_SUBSCRIPTION_SORT_BY_KEYS',
-  'LearnerDashboardBackendApiService', 'UrlInterpolationService',
-  'LEARNER_DASHBOARD_SECTIONS', 'LEARNER_DASHBOARD_SUBSECTIONS',
-  'threadStatusDisplayService', 'oppiaDatetimeFormatter',
-  'FEEDBACK_THREADS_SORT_BY_KEYS',
-  'HUMAN_READABLE_FEEDBACK_THREADS_SORT_BY_KEYS',
+  'EXPLORATIONS_SORT_BY_KEYS_AND_I18N_IDS',
+  'SUBSCRIPTION_SORT_BY_KEYS_AND_I18N_IDS',
+  'FATAL_ERROR_CODES', 'LearnerDashboardBackendApiService',
+  'UrlInterpolationService', 'LEARNER_DASHBOARD_SECTION_I18N_IDS',
+  'LEARNER_DASHBOARD_SUBSECTION_I18N_IDS', 'threadStatusDisplayService',
+  'oppiaDatetimeFormatter', 'FEEDBACK_THREADS_SORT_BY_KEYS_AND_I18N_IDS',
+  'FeedbackThreadObjectFactory', 'FeedbackMessageObjectFactory',
   function(
-      $scope, $rootScope, $window, $http, $modal, EXPLORATIONS_SORT_BY_KEYS,
-      SUBSCRIPTION_SORT_BY_KEYS, HUMAN_READABLE_EXPLORATIONS_SORT_BY_KEYS, 
-      FATAL_ERROR_CODES, HUMAN_READABLE_SUBSCRIPTION_SORT_BY_KEYS,
+      $scope, $rootScope, $window, $http, $modal,
+      EXPLORATIONS_SORT_BY_KEYS_AND_I18N_IDS,
+      SUBSCRIPTION_SORT_BY_KEYS_AND_I18N_IDS, FATAL_ERROR_CODES,
```

Comment:
Nit: have the line breaks match those in the imports above. Easier to compare and make sure nothing's been left out by accident (otherwise minification would fail).